apisix/plugins/authz-keycloak.lua

```diff
@@ -695,8 +703,107 @@ local function fetch_jwt_token(ctx)
     return token
 end
 
+-- To get new access token by calling get token api
+local function generate_token_using_password_grant(conf,ctx)
+    log.warn("generate_token_using_password_grant Function Called")
```

Comment:
Why should we use `warn` level ?

Reply:
Agree... it should be debug level log. I changed it.